atlp-rwanda/e-comm-team-brogrammers-fn
# #184948165 User gives reviews to products

#184948165 a seller delete item and update availability
This PR is for user reviews , feedback and ratings
#### What does this PR do?

this Pr is for user to be able to leave review for product,update review and even delete the review

#### Description of Task to be completed?

user should be able to leave review for product, update review and even delete the review if he us logged in if not logged in you should not review

#### How should this be manually tested?


-from list of all products click on product name and you will be directed to one product Display and below you will see reviews
- login to be able to submit review,delete it or even update it

#### Any background context you want to provide?

#### What are the relevant pivotal tracker/Trello stories?

https://www.pivotaltracker.com/story/show/184948182
#### Screenshots (if appropriate)
<img width="1096" alt="Screenshot 2023-05-22 at 07 39 34" src="https://github.com/atlp-rwanda/e-comm-team-brogrammers-fn/assets/99663528/81a95dde-db39-4bae-9d03-1852391e6832">
<img width="930" alt="Screenshot 2023-05-22 at 07 39 41" src="https://github.com/atlp-rwanda/e-comm-team-brogrammers-fn/assets/99663528/979a3afb-5450-44a9-8e8f-cd4dc05ddad1">

#### Questions:
